snippets/olmv1-known-issue-service-accounts.adoc

```diff
@@ -0,0 +1,14 @@
+// Text snippet included in the following modules:
+//
+// * modules/olmv1-installing-an-operator.adoc
+// * release_notes/ocp-4-17-release-notes.adoc (enterprise-4.17 branch only)
+
+:_mod-docs-content-type: SNIPPET
+
+There is a known issue in {olmv1}. If you do not assign the correct role-based access controls (RBAC) to an extension's service account, {olmv1} gets stuck in an reconciliation loop.
+
+Currently, {olmv1} does not have tools to help customers find the correct RBAC for a service account.
+
+Since {olmv1} is a Technology Preview feature and must not be used on production clusters, you can avoid this issue by using the more permissive set of RBAC included in the documentation.
```

Comment:
```suggestion
Because {olmv1} is a Technology Preview feature and must not be used on production clusters, you can avoid this issue by using the more permissive set of RBAC included in the documentation.
```